src/test/java/dp/xlsx/GroupTest.java

```diff
@@ -1,55 +1,73 @@
             add(new DimensionData(DimensionType.OTHER, "19", "19"));
             add(new DimensionData(DimensionType.OTHER, "Indirect taxes", "indirect-taxes"));
             add(new DimensionData(DimensionType.OTHER, "1980s", "1980s"));
-        }};
+        }}, null);
 
-        Group groupOne = new Group(rowOne, null);
-        Group groupTwo = new Group(rowTwo, null);
+        clothing = new Group(new ArrayList<DimensionData>() {{
+            add(new DimensionData(DimensionType.GEOGRAPHY, "K02000001", "K02000001"));
+            add(new DimensionData(DimensionType.OTHER, "cpi1dim1G30100", "03.1 Clothing"));
+        }}, null);
 
-        assertThat(groupOne.hashCode()).isNotEqualTo(groupTwo.hashCode());
+        alcoholicBeverages = new Group(new ArrayList<DimensionData>() {{
+            add(new DimensionData(DimensionType.GEOGRAPHY, "K02000001", "K02000001"));
+            add(new DimensionData(DimensionType.OTHER, "cpi1dim1G20100", "02.1 Alcoholic beverages"));
+        }}, null);
     }
 
     @Test
-    public void test_hashCode_sameDimensions_shouldHaveTheSameCode() throws Exception {
-        List<DimensionData> rowOne = new ArrayList<DimensionData>() {{
-            add(new DimensionData(DimensionType.GEOGRAPHY, "United Kingdom", "K02000001"));
-            add(new DimensionData(DimensionType.OTHER, "18", "18"));
-            add(new DimensionData(DimensionType.OTHER, "Indirect taxes", "indirect-taxes"));
-            add(new DimensionData(DimensionType.OTHER, "1990s", "1990s"));
-        }};
+    public void testEquals_verySimilarDimensions_shouldNotBeEqual() throws Exception {
+        assertThat(indirectTaxesAge18The90s).isNotEqualTo(indirectTaxesAge19The80s);
+    }
 
-        List<DimensionData> rowTwo = new ArrayList<DimensionData>() {{
-            add(new DimensionData(DimensionType.GEOGRAPHY, "United Kingdom", "K02000001"));
-            add(new DimensionData(DimensionType.OTHER, "18", "18"));
-            add(new DimensionData(DimensionType.OTHER, "Indirect taxes", "indirect-taxes"));
-            add(new DimensionData(DimensionType.OTHER, "1990s", "1990s"));
-        }};
+    @Test
+    public void testEquals_twoObjectWithSameValues_shouldBeEqual() {
+        Group dupplicate = new Group(new ArrayList<DimensionData>() {{
+            add(new DimensionData(DimensionType.GEOGRAPHY, "K02000001", "K02000001"));
+            add(new DimensionData(DimensionType.OTHER, "cpi1dim1G20100", "02.1 Alcoholic beverages"));
+        }}, null);
+
+        assertThat(alcoholicBeverages).isEqualTo(dupplicate);
+    }
 
-        Group groupOne = new Group(rowOne, null);
-        Group groupTwo = new Group(rowTwo, null);
+    @Test
+    public void testEquals_twoObjectWithDifferentValues_shouldNotBeEqual() {
+        assertThat(clothing).isNotEqualTo(alcoholicBeverages);
+    }
+
+    @Test
+    public void testHashCode_twoEqualObjects_shouldProduceEqualHashcode() {
```

Comment:
This test appears to be the same as `testEquals_twoObjectWithSameValues_shouldBeEqual`. This test should be comparing the hash codes?